src/odin/enhancedtrippath.cc

```diff
@@ -477,7 +477,7 @@ float EnhancedTripLeg_Edge::GetLength(const Options::Units& units) {
 
 bool EnhancedTripLeg_Edge::HasActiveTurnLane() const {
   for (const auto& turn_lane : turn_lanes()) {
-    if (turn_lane.is_active()) {
+    if (turn_lane.state() == ::valhalla::TurnLane::kActive) {
```

Comment:
you could do the shorter `TurnLane::kActive`

Reply:
yes! I forgot that we're already within the valhalla namespace here! thx.